### PR TITLE
Make some design changes

### PR DESCRIPTION
**Pull request by elsteelbrain**
Just experimenting with the current design. PR for review and comments.

### CODE REVIEW

**Comment by DavidWells** [comment]
Deploy preview ready!

Built with commit 4bdcce0eded85e6149aca56a8ceb5535b52decd5

https://deploy-preview-145--serverless.netlify.com

**Comment by jfdesroches** [comment]
I dig this, pretty clean! Sad to see the homepage header image go, it looks a little empty without it (but not bad, let me see if I get used to it). 

A few tiny things: 
- I like the white Gateway card on the homepage, but am curious if the icon would look good with the blue background (this way we're visually tying blue to the EG up front)
- in terms of CTAs (calls to action) on the homepage, we should decide what we want people to do. Is it check out the docs? Is it sign up for Beta? Having multiple is fine so long as we are ok with them, and can track what a "conversion" means. CC @DavidWells and @worldsoup on this
- the hover state of the nav works well with a white background, but kinda fades away on the darker ones.

Otherwise, overall everything looks good, thanks for taking the initiative on this.

File: src/components/Terminal/Terminal.js [hunk 1]
```diff
@@ -84,10 +85,11 @@ export default class TerminalCommands extends Component {
     return output
   }
   render() {
+    const { className } = this.props
     const terminalOutput = this.renderLines()
 
     return (
-      <div className={styles.wrapper}>
```
Comment (DavidWells) [inline src/components/Terminal/Terminal.js]: We have a conflict here.

https://github.com/serverless/site/blob/master/src/components/Terminal/Terminal.js#L99

Rename my prop to `terminalClassName` (and maybe yours to  `wrapperClassName`)

Comment (elsteelbrain) [reply src/components/Terminal/Terminal.js]: I'll remove this altogether. Not using it currently.

**Pull request merged**